packages/deployment/Makefile

```diff
@@ -7,33 +7,35 @@ GIT_REVISION := $(shell hash=$$(git rev-parse --short HEAD); \
 	dirty=`git diff --quiet 2>/dev/null || echo -dirty`; \
 	echo "$$hash$$dirty")
 
+DOCKER_BUILD := docker build
```

Comment:
Thanks, I think it's easy enough to tailor locally when needed.

Out of curiosity, what cloud are you using? We have a Depot.dev account now and it can take over buildx without tailoring CLI. `brew install depot && depot configure-docker`

Reply:
Oh, at the time I was playing with Docker buildcloud, just for funsies.